ImprovedAnvil/lib/core/2da.tph

```diff
@@ -241,12 +274,31 @@ COPY_EXISTING ~weapprof.2da~  ~override/weapprof.2da~
   SET_2DA_ENTRIES_NOW ~_#_#_#weapprof~ ~colcount~
   PRETTY_PRINT_2DA
 
+// change Sword and Shield description
+COPY_EXISTING ~weapprof.2da~  ~override/weapprof.2da~
+  COUNT_2DA_COLS ~colcount~
+  SET_2DA_ENTRY 29 3 ~colcount~ RESOLVE_STR_REF (@12087)
+  PRETTY_PRINT_2DA
+
+// enable ** in Sword and Shield for Cleric and all kits and *** for Tyr
+COPY_EXISTING ~weapprof.2da~  ~override/weapprof.2da~
+  COUNT_2DA_COLS ~colcount~
+  SET_2DA_ENTRY_LATER ~_#_#_#weapprof~ 29 6 ~2~
+  SET_2DA_ENTRY_LATER ~_#_#_#weapprof~ 29 48 ~2~
+  SET_2DA_ENTRY_LATER ~_#_#_#weapprof~ 29 49 ~2~
+  SET_2DA_ENTRY_LATER ~_#_#_#weapprof~ 29 50 ~2~
+  SET_2DA_ENTRY_LATER ~_#_#_#weapprof~ 29 62 ~3~
+  SET_2DA_ENTRY_LATER ~_#_#_#weapprof~ 29 63 ~2~
+  SET_2DA_ENTRIES_NOW ~_#_#_#weapprof~ ~colcount~
+  PRETTY_PRINT_2DA
+
 // add records for the new portrait icons
 COPY_EXISTING ~statdesc.2da~ ~override/statdesc.2da~
   COUNT_2DA_COLS ~colcount~
   COUNT_2DA_ROWS ~colcount~ ~rowcount~
 
   SET_2DA_ENTRY 125 2 ~colcount~ ~SPINI27D~
+  SET_2DA_ENTRY 66 1 ~colcount~ RESOLVE_STR_REF (@12095)
```

Comment:
why does 66 go after 125? Stylistically better if they located in natural order towards to one another

Reply:
Swapped them